app-text/texlive/texlive-2021.recipe

```diff
@@ -6,27 +6,35 @@ are free software, including support for many languages around the world."
 HOMEPAGE="http://tug.org/texlive/"
 COPYRIGHT="1996 by collaboration between the TeX user groups."
 LICENSE="GNU GPL v2"
-REVISION="3"
-SOURCE_URI="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-source.tar.xz"
-SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-texmf.tar.xz"
-CHECKSUM_SHA256="08bb4eb02923ac65ce1183612c6b64da185904f26a1660f5ca4f902d055795be"
-CHECKSUM_SHA256_2="e0227251c5aa23f8b88973c298cc80b03bd238c645f7f82a6ce888a06b295963"
-SOURCE_DIR="texlive-20140525-source"
-SOURCE_DIR_2="texlive-20140525-texmf"
-PATCHES="texlive-2014.patchset"
-ADDITIONAL_FILES="tlutils.pm.patch"
+REVISION="1"
+fullVersion=20210325
+SOURCE_URI="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-source.tar.xz"
+SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-texmf.tar.xz"
+SOURCE_URI_3="ftp://tug.org/historic/systems/texlive/$portVersion/install-tl-unx.tar.gz"
+SOURCE_URI_4="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-tlpdb-full.tar.gz"
+CHECKSUM_SHA256="7aefd96608d72061970f2d73f275be5648ea8ae815af073016d3106acc0d584b"
+CHECKSUM_SHA256_2="ff12d436c23e99fb30aad55924266104356847eb0238c193e839c150d9670f1c"
+CHECKSUM_SHA256_3="74eac0855e1e40c8db4f28b24ef354bd7263c1f76031bdc02b52156b572b7a1d"
+CHECKSUM_SHA256_4="21d218dcf56160081e8c376ec9d6b4cfc750e2a0fea14ee24884b1d5039f2566"
+SOURCE_DIR="texlive-$fullVersion-source"
+SOURCE_DIR_2="texlive-$fullVersion-texmf"
+SOURCE_DIR_3="install-tl-20210324"
+SOURCE_DIR_4=""
+PATCHES="texlive-$portVersion.patchset"
+ADDITIONAL_FILES="texlive.profile"
 
```

Comment:
Shouldn't we add:
```
commandBinDir=$binDir
commandSuffix=$secondaryArchSuffix
if [ "$targetArchitecture" = x86_gcc2 ]
then
	commandBinDir=$prefix/bin
	commandSuffix=
fi
```
here?

Reply:
If yes, we should add `$commandSuffix` to every provides and the `configure` invocation should be also adjusted with `$commandBinDir`.